recipes/qt/6.x.x/conanfile.py

```diff
@@ -554,7 +556,7 @@ def _create_module(module, requires=[]):
             self.cpp_info.components[componentname].requires = _get_corrected_reqs(requires)
 
         def _create_plugin(pluginname, libname, type, requires):
-            componentname = "qt%s" % pluginname
+            componentname = "Qt6%s" % pluginname
```

Comment:
Why this modification? It's a breaking change because downstream recipes might already rely on these names, it's like changing the name of a recipe.

Reply:
As this is how Qt names its libraries. It avoids some parsing and keeps it consistent, buuuuuuuut fair point will change.

Reply:
Fixed